orderer/common/cluster/util.go

```diff
@@ -942,3 +945,9 @@ func LatestHeightAndEndpoint(puller ChainPuller) (string, uint64, error) {
 	}
 	return mostUpToDateEndpoint, maxHeight, nil
 }
+
+func EncodeTimestamp(t *timestamp.Timestamp) []byte {
+	b := make([]byte, 8)
+	binary.LittleEndian.PutUint64(b, uint64(t.Seconds))
+	return b
```

Comment:
I think you have to add the Nanos as well, append the 4 bytes to the 8 bytes Seconds to make a 12 byte slice.